scripts/helpers/function-helpers.ts

```diff
@@ -111,7 +115,29 @@ export async function democracyWrapper(
       }
 
       console.log("--> Democracy Tx sent\n");
-    } else if (proposalType == "council-external") {
+    }
+    else if (proposalType == "v2") {
+      let t = {
+        root: { system: 'Root' },
+        whitelisted: { Origins: 'WhitelistedCaller' } ,
+        general: { Origins: 'GeneralAdmin' },
+        canceller: { Origins: 'ReferendumCanceller' },
+        killer: { Origins: 'ReferendumKiller' }
+      }[track];
+      
+      if(t == undefined) {
+        t = JSON.parse(track);
```

Comment:
I understand you are allowing the input to be whatever json object. But we should provide an example